guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,62 @@
+[[migrating_pulp_content]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content.
+For large systems, this can mean several days of downtime.
+
+Use this procedure to begin migrating content from Pulp 2 to Pulp 3.
+
+.Procedure
+
+. View details of the content you are migrating using the following command:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-maintain} content migration-stats
+----
++
+Use this command as often as necessary during the migration process to determine how long the process will take.
+It also identifies corrupted or missing content that might cause the migration to fail.
+Output is similar to the following:
```

Comment:
I wonder if this doc should talk about what to do with corrupt/missing content and the workflow around that?  It was discussed a bit more indepth in the recent CEE enablement session.  I'm happy to provide more details as well

Reply:
Possibly at GA. I think it unecessary at beta.

Reply:
If it's necessary we can add it at GA. I don't think it's necessary at beta.